parquet/src/main/java/org/apache/iceberg/parquet/ParquetReader.java

```diff
@@ -34,7 +35,7 @@
 import org.apache.parquet.hadoop.ParquetFileReader;
 import org.apache.parquet.schema.MessageType;
 
-public class ParquetReader<T> extends CloseableGroup implements CloseableIterable<T> {
+public class ParquetReader<T> extends CloseableGroup implements InternalData.DataIterable<T> {
```

Comment:
Changes in the Parquet should be in another PR? as the PR title says just replace Avro generic with Avro internal data?

Reply:
Added comment about the parquet to the description.